bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx142.html

```diff
@@ -0,0 +1,193 @@
+            </div>
+            <h3 class="wnp-subtitle">Private tabs that stay private</h3>
+            <p class="wnp-body">Your private tabs lock automatically when you step away — and only unlock with your face, fingerprint, or PIN.</p>
+          </article>
+          <article class="wnp-card">
+            <div>
+              <span class="wnp-tag wnp-tag-blue"><img src="/media/img/firefox/whatsnew/whatsnew142/language-icon.svg" width="12" height="12" alt="" aria-hidden="true" />Language</span><span class="wnp-tag wnp-tag-android"><img src="/media/img/firefox/whatsnew/whatsnew142/android-icon.png" width="12" height="12" alt="" aria-hidden="true" />Android</span>
+            </div>
+            <h3 class="wnp-subtitle">Getting even more multilingual</h3>
+            <p class="wnp-body">Now translate web pages into Japanese, Chinese, Korean and more, so you can browse in your preferred language.</p>
+          </article>
+          <article class="wnp-card">
+            <div>
+              <span class="wnp-tag wnp-tag-orange"><img src="/media/img/firefox/whatsnew/whatsnew142/shield.svg" width="12" height="12" alt="" aria-hidden="true" />Security</span><span class="wnp-tag wnp-tag-ios"><img src="/media/img/firefox/whatsnew/whatsnew142/apple-icon.png" width="12" height="12" alt="" aria-hidden="true" />iOS</span>
+            </div>
+            <h3 class="wnp-subtitle">Smarter passwords, fewer hassles</h3>
+            <p class="wnp-body">Firefox suggests strong passwords when you’re creating a new account on any site, and keeps them secure, ready on any device when you sync.</p>
+          </article>
+          <article class="wnp-card">
+            <div>
+              <span class="wnp-tag wnp-tag-yellow"><img src="/media/img/firefox/whatsnew/whatsnew142/design-icon.svg" width="12" height="12" alt="" aria-hidden="true" />Design</span><span class="wnp-tag wnp-tag-ios"><img src="/media/img/firefox/whatsnew/whatsnew142/apple-icon.png" width="12" height="12" alt="" aria-hidden="true" />iOS</span>
+            </div>
+            <h3 class="wnp-subtitle">A cleaner look with sharper focus</h3>
+            <p class="wnp-body">A streamlined UI and upgraded dark mode on Firefox for iOS bring clarity and calm to everything you browse.</p>
+          </article>
+        </div>
+      </div>
+    </section>
+
+    <!-- QR Code Banner -->
+    <section class="wnp-section wnp-section-mobile" aria-labelledby="wnp-qr-qr-heading">
+      <div class="wnp-container">
+        <div class="wnp-qr">
+          <div class="wnp-qr-layout">
+            <div class="wnp-qr-media" aria-hidden="true">
+              <div class="wnp-qr-qr" aria-label="QR code to download Firefox Mobile" role="img">{{ qrcode("https://www.firefox.com/browsers/mobile/app/?product=firefox&campaign=firefox-all", 12) }}</div>
```

Comment:
FYI the campaign here will need to be updated (this one is apparently taken from fx/download/all mobile flow), so just a reminder to get the correct link for the QR code.